src/components/Rule/Rule.jsx

```diff
@@ -1,39 +1,33 @@
-import cx from 'classnames';
-import React from 'react';
-import PropTypes from 'prop-types';
-import style from './style.scss';
-import {
-  getDataAttrs,
-  getDataProps,
-} from '../../dataUtils';
+import cx from "classnames";
+import React from "react";
+import PropTypes from "prop-types";
+import style from "./style.scss";
+import { getDataAttrs, getDataProps } from "../../dataUtils";
 
-const type = { SOLID: 'solid', DOTTED: 'dotted', DASHED: 'dashed' };
+const type = { SOLID: "solid", DOTTED: "dotted", DASHED: "dashed" };
 
 class Rule extends React.Component {
   render() {
     const ruleStyle = cx(style.wrapper, {
       [style.dotted]: this.props.type === type.DOTTED,
       [style.dashed]: this.props.type === type.DASHED,
+      [style.withTopMargin]: this.props.withTopMargin
```

Comment:
added the ability to specify a top margin (all my mocks have both a top and bottom margin around the Rule